### PR TITLE
added dalex integration

### PR DESCRIPTION
**Pull request by jakubczakon**
Added:
* helpers for logging DALEX explanations as interactive charts
* helpers to log DALEX explainer object as pickle

### CODE REVIEW

File: docs/integrations/dalex.rst [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+This integration enables you to log |DALEX| global and local explanations as well as the ``dalex.expainer`` object to Neptune.
+
+.. image:: ../_static/images/others/dalex_neptuneai.png
+   :target: ../_static/images/others/dalex_neptuneai.png
+   :alt: DALEX Neptune integration
+
+Requirements
+------------
+To use Neptune + DALEX integration you need to have installed is |neptune-client| and |neptune-contrib|.
```
Comment (aniezurawski) [inline docs/integrations/dalex.rst:12]: `you need to have installed is` -> `you need to have installed`

File: docs/integrations/dalex.rst [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+
+Log local explanations
+----------------------
+
+To log local explanations you need to pass the observation and the explainer object.
+
+The following charts can be logged to Neptune:
+
+* break down
+* break down with interactions,
```
Comment (aniezurawski) [inline docs/integrations/dalex.rst:76]: Unnecessary comma.

File: docs/integrations/dalex.rst (hunk 1 above)
Comment (jakubczakon) [reply docs/integrations/dalex.rst]: done

File: docs/integrations/dalex.rst (hunk 2 above)
Comment (jakubczakon) [reply docs/integrations/dalex.rst]: done

**Pull request merged**